Makefile

```diff
@@ -331,14 +330,15 @@ importer-image: PLATFORMS=linux/amd64
 importer-image: PUSH=--load
 importer-image: importer-image-build
 
-
 # Build the kueue-viz dashboard images (frontend and backend)
 .PHONY: kueue-viz-image-build
 kueue-viz-image-build:
+	docker run --privileged --rm tonistiigi/binfmt --install all
```

Comment:
This is not obvious to me. Can you add a comment that this is required for emulated builds for docker?

Reply:
Yes for sure, I'm not even sure this is the best place for this. I put it in quickly because I was having trouble reproducing on my local env.

/hold

Reply:
/unhold